lib/bap_ida/bap_ida.mli

```diff
@@ -26,7 +26,7 @@ module Std : sig
 
     (** [create target] create an IDA instance that will work with
         [target] executable. *)
-    val create : string -> t
```

Comment:
Seems like you committed this file's changes by mistake. We were using this file for prototyping earlier.